keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -51,14 +51,16 @@ private fun ClusterSpec.resolveLaunchConfiguration(region: SubnetAwareRegionSpec
     instanceType = checkNotNull(overrides[region.name]?.launchConfiguration?.instanceType
       ?: defaults.launchConfiguration?.instanceType),
     ebsOptimized = checkNotNull(overrides[region.name]?.launchConfiguration?.ebsOptimized
-      ?: defaults.launchConfiguration?.ebsOptimized),
+      ?: defaults.launchConfiguration?.ebsOptimized
+      ?: LaunchConfiguration.DEFAULT_EBS_OPTIMIZED),
```

Comment:
This fixes an unrelated issue I caught when attempting to use the diff API on an exported spec. Since we're nulling out fields with defaults now, a spec with a missing `ebsOptimized` field is a valid one.